paasta_tools/vitesscluster_tools.py

```diff
@@ -15,6 +17,594 @@
+                "key": "vault-vttablet-approle-roleid",
+            }
+        },
+    },
+    {
+        "name": "VAULT_SECRETID",
+        "valueFrom": {
+            "secretKeyRef": {
+                "name": "paasta-vitessclusters-secret-vitess-k8s-vault-vttablet-approle-secretid",
+                "key": "vault-vttablet-approle-secretid",
+            }
+        },
+    },
+    {
+        "name": "VAULT_CACERT",
+        "value": "/etc/vault/all_cas/acm-privateca-uswest1-devc.crt",
+    },
+]
+
+# Vault auth related variables
+VTGATE_EXTRA_ENV = [
+    {
+        "name": "VAULT_ADDR",
+        "value": "https://vault-dre.uswest1-devc.yelpcorp.com:8200",
+    },
+    {
+        "name": "VAULT_ROLEID",
+        "valueFrom": {
+            "secretKeyRef": {
+                "name": "paasta-vitessclusters-secret-vitess-k8s-vault-vtgate-approle-roleid",
+                "key": "vault-vtgate-approle-roleid",
+            }
+        },
+    },
+    {
+        "name": "VAULT_SECRETID",
+        "valueFrom": {
+            "secretKeyRef": {
+                "name": "paasta-vitessclusters-secret-vitess-k8s-vault-vtgate-approle-secretid",
+                "key": "vault-vtgate-approle-secretid",
+            }
+        },
+    },
+    {
+        "name": "VAULT_CACERT",
+        "value": "/etc/vault/all_cas/acm-privateca-uswest1-devc.crt",
+    },
+]
+
+
+# Extra Flags
+VTADMIN_EXTRA_FLAGS = {"grpc-allow-reflection": "true"}
+
+VTCTLD_EXTRA_FLAGS = {
+    "disable_active_reparents": "true",
+    "security_policy": "read-only",
+}
+
+VTTABLET_EXTRA_FLAGS = {
+    "log_err_stacks": "true",
+    "grpc_max_message_size": "134217728",
+    "init_tablet_type": "replica",
+    "queryserver-config-schema-reload-time": "1800",
+    "dba_pool_size": "4",
+    "vreplication_heartbeat_update_interval": "60",
+    "vreplication_tablet_type": "REPLICA",
+    "keep_logs": "72h",
+    "enable-lag-throttler": "true",
+    "throttle_check_as_check_self": "true",
+    "throttle_metrics_query": "",
+    "throttle_metrics_threshold": "",
+    "db_charset": "utf8mb4",
+    "disable_active_reparents": "true",
+}
+
+
+def build_affinity_spec(paasta_pool):
+    spec = {
+        "nodeAffinity": {
+            "requiredDuringSchedulingIgnoredDuringExecution": {
+                "nodeSelectorTerms": [
+                    {
+                        "matchExpressions": [
+                            {
+                                "key": "yelp.com/pool",
+                                "operator": "In",
+                                "values": [paasta_pool],
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    }
+    return spec
+
+
+def build_extra_labels(paasta_pool, paasta_cluster):
+    """
+    Build extra labels to adhere to paasta contract
+    """
+    extra_labels = {
+        "yelp.com/owner": "dre_mysql_working_hours",
```

Comment:
fwiw: we don't page on these - it's just for ee-metrics use: so it'd probably be "better" to have this be `dre_mysql` (assuming y'all have a sensu team called dre_mysql)

Reply:
https://github.com/Yelp/paasta/pull/3889/commits/c7a6b7f7665b11eb4e812cba4a46457b851040cd

Reply:
This still needs fixing, has default to compute-infra when used from parent class 
```
'extraLabels': {'yelp.com/owner': 'compute_infra_platform_experience' ...
```

Reply:
https://github.com/Yelp/paasta/pull/3889/commits/700065a7d9b82a3facfefb9e6a2f79e508b96dd7